src/test/net_peer_connection_tests.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2023-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chainparams.h>
+#include <compat/compat.h>
```

Comment:
this seems to be failing in the Win64 CI 

```
C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\src\test\net_peer_connection_tests.cpp(6,10): fatal error C1083: Cannot open include file: 'compat/compat.h': No such file or directory [C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\build_msvc\test_bitcoin\test_bitcoin.vcxproj]
``` 

https://cirrus-ci.com/task/4941676805881856?logs=build#L2242